myindex.html

```diff
@@ -0,0 +1,380 @@
+                    facilisi. Nulla egestas at nunc at volutpat. Nullam sodales at diam vel tempor. Sed pretium, leo vel pellentesque dictum, ligula […]</p>
+            </div>
+        </div>
+
+        <div class="col-2 "></div>
+    </div>
+
+    </div>
+
+
+    <!-- Footer -->
+    <hr class="bhr"></hr>
+    <div class="row footerContainer">
+        <div class="insiderow">
+
+            <div class="row row-cols-1 row-cols-md-4">
+                <div class="col mb-4">
+                    <div class="card h-100">
+                        <div class="card-body">
+                            <h5 class="card-title">REVERA</h5>
+                            <p class="card-text">Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum vehicula erat non congue cursus. Aenean auctor nulla quis augue dictum, sed sagittis odio varius? Aliquam nec venenatis turpis.
+                                Aenean euismod arcu vel volutpat tristique. In mattis ipsum quis magna feugiat, id tempus metus accumsan.</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="col mb-4">
+                    <div class="card h-100">
+                        <div class="card-body">
+                            <h5 class="card-title">META</h5>
+                            <p class="card-text">Log in.</p>
+                            <p class="card-text">Entries RSS</p>
+                            <p class="card-text">Comments RSS</p>
+                            <p class="card-text">CWordPress.org</p>
+
+
+                        </div>
+                    </div>
+                </div>
+                <div class="col mb-4">
+                    <div class="card h-100">
+                        <div class="card-body">
+                            <h5 class="card-title">ARCHIVES</h5>
+                            <p class="card-text">August 2013</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="col mb-4">
+                    <div class="card h-100">
+                        <div class="card-body">
+                            <h5 class="card-title">RECENT POSTS
+                            </h5>
+                            <p class="card-text">Etiam mauris tortor, pharetra quis lobortis in, pharetra in diam</p>
+
+                            <p class="card-text">Nunc libero elit, iaculis nec augue non</p>
+
+                            <p class="card-text">Falauris sollicitudin quis dolor venenatis facilisis</p>
+                            <p class="card-text">Phasellus aliquet iaculis neque accumsan aliquam</p>
+                            <p class="card-text">Vivamus blandit ipsum sed arcu euismod</p>
+
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+    </div>
+
```

Comment:
The bottom bar, containing the copyright message in the example, seems to be missing.